src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/LegacyPbaAccountServiceImplTest.java

```diff
@@ -50,28 +43,9 @@ public void setup() {
 
     @Test
     public void testFindLegacyAccountByUserEmailWhenPbaIsEmpty() {
-        when(configurationMock.getPbaFromUserAccountMap()).thenReturn("false");
-
         paymentAccountPbaNumbers = sut.findLegacyPbaAccountByUserEmail(professionalUser);
         assertThat(paymentAccountPbaNumbers).isNotNull();
         assertThat(paymentAccountPbaNumbers.size()).isEqualTo(1);
-        assertThat(paymentAccountPbaNumbers.get(0)).isEqualTo("PBA123");
-        verify(configurationMock, times(1)).getPbaFromUserAccountMap();
-    }
-
-    @Test
-    public void testFindLegacyAccountByUserEmail() throws Exception {
-        when(configurationMock.getPbaFromUserAccountMap()).thenReturn("true");
-        List<UserAccountMap> userAccountMapData = new ArrayList<>();
-        UserAccountMapId newUserAccountMapId = new UserAccountMapId(professionalUser, paymentAccount);
-        UserAccountMap userAccountMap = new UserAccountMap(newUserAccountMapId);
-        userAccountMapData.add(userAccountMap);
-
-        paymentAccountPbaNumbers = sut.findLegacyPbaAccountByUserEmail(professionalUser);
-
-        assertThat(paymentAccountPbaNumbers).isNotNull();
-        assertThat(paymentAccountPbaNumbers.size()).isGreaterThanOrEqualTo(0);
-
-        verify(configurationMock, times(2)).getPbaFromUserAccountMap();
```

Comment:
Any reason this test was deleted - seems like a test we should have for the Legacy endpoint?

Reply:
true condition is not required at the moment.